onnxruntime/test/shared_lib/test_io_types.cc

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
+#if !defined(ORT_MINIMAL_BUILD)
```

Comment:
This is to fix a warning in "build_x64_minimal_no_exception" build job:

onnxruntime\test\shared_lib\test_io_types.cc(10,13): Warning C4505: 'TestModelInfo': unreferenced function with internal linkage has been removed